src/Modules/OrchardCore.Commerce.Payment.Stripe/Views/StripeApiSettings.Edit.cshtml

```diff
@@ -16,6 +16,10 @@
           hint="@(new HtmlString(" ").Join(T["Your Stripe Secret API key."], emptyAfterSaving))"
           type="password"></fieldset>
 
+<fieldset asp-for="AccountId"
+          label="@T["Account Id"]"
+          hint="@T["Your Stripe Connect Account Id. This is necessary when using a Stripe connected account."]"></fieldset>
```

Comment:
In frontend text the term "ID" is all caps because it's an acronym.
```suggestion
          label="@T["Account ID"]"
          hint="@T["Your Stripe Connect Account ID. This is necessary when using a Stripe connected account."]"></fieldset>
```